dashboards-observability/public/components/visualizations/charts/helpers/viz_types.ts

```diff
@@ -109,20 +109,31 @@ const defaultUserConfigs = (queryString, visualizationName: string) => {
       },
     };
     if (visualizationName === VIS_CHART_TYPES.LogsView) {
-      tempUserConfigs = {
-        ...tempUserConfigs,
-        [AGGREGATIONS]: [],
-        [GROUPBY]: statsTokens.aggregations
+      const dimensions = [
+        ...statsTokens.aggregations
           .map((agg) => ({
-            label: agg.name ?? '',
-            name: agg.name ?? '',
+            label: `${agg.function.name}(${agg.function.value_expression})` ?? '',
+            name: `${agg.function.name}(${agg.function.value_expression})` ?? '',
```

Comment:
Similar to above comment, this can also be saved in a variable.

Reply:
Resolved.